src/main/ApplicationImpl.cpp

```diff
@@ -679,9 +679,9 @@ ApplicationImpl::syncOwnMetrics()
     mMetrics->NewCounter({"process", "memory", "handles"})
         .set_count(mProcessManager->getNumRunningProcesses());
 
-    // Update ioservice related metrics
-    mMetrics->NewCounter({"process", "ioservice", "queue"})
-        .set_count(static_cast<int64_t>(getClock().getExecutionQueueSize()));
+    // Update action-queue related metrics
```

Comment:
Should we add a metric that tracks how "overloaded" we are (we have `recentIdleCrankPercent`)?

Maybe related - right now `recentIdleCrankPercent` doesn't care if the scheduler is overloaded... Maybe it should?

Reply:
I'll add a metric here, yeah.

I would love to redo the code in `overlay/LoadManager.cpp` and/or connect it too this new load signal, but I feel like this PR is already getting pretty overgrown; I'll file a followup bug.